app/assets/stylesheets/syntax.scss

```diff
@@ -13,214 +13,220 @@ div.inner-comment div.body div.highlight pre.highlight {
   }
 }
 
+$code-highlight-background-color: #29292e;
+$code-highlight-text-color: #89c4f4;
+$code-highlight-comment-color: #d5d5d5;
```

Comment:
I think it's common pattern in code editors to make comments color a bit "faded out". Previously it was grey `#75715e` which I think worked pretty well.

Reply:
@ludwiczakpawel I cannot use the original color as it has terrible accessibility contrast ratio. I used very similar one, but slightly more visible on the background.

<img width="949" alt="Screenshot 2020-02-29 at 12 18 08" src="https://user-images.githubusercontent.com/3818442/75606577-6b788c00-5aee-11ea-96eb-ea21eaa6d588.png">